src/composable/types.ts

```diff
@@ -89,22 +89,16 @@ export type OwnerContext = {
 }
 
 export type PollParams = OwnerContext & {
+  orderBook: OrderBookApi
```

Comment:
I see you want to simplify the name, but this is basically the REST API, and we call it everywhere `orderBookApi`

IMO, `orderBook` could be how we name an instance of the orderbook, not the facade to call the rest API

Reply:
Reverted it back to `orderBookApi`